### PR TITLE
Add analytics-extensions

### PR DESCRIPTION
**Pull request by ivantha**
### Purpose
Add support for Google Analytics and MixPanel.

### CODE REVIEW

File: analytics-extensions/analytics.jsp [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+--%>
+
+<%!
+    private static final String GOOGLE_ANALYTICS_TOKEN = null;
```
Comment (JKAUSHALYA) [inline analytics-extensions/analytics.jsp:20]: Shall we add a comment here to say "add the token that is retrieved from the google"?

File: analytics-extensions/analytics.jsp [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    private static final String MIXPANEL_TOKEN = null;
```
Comment (JKAUSHALYA) [inline analytics-extensions/analytics.jsp:21]: Similar comment like above for here as well.

File: analytics-extensions/analytics.jsp (hunk 1 above)
Comment (mefarazath) [reply analytics-extensions/analytics.jsp:20]: How about moving these two configs to a separate JSP like config.jsp and include it here :)

File: analytics-extensions/analytics.jsp [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        });
+
+        $(window).on("beforeunload", function () {
+            if(!eventFired) {
+                fireEvent("left")
+            }
+        });
+    </script>
+</head>
+</html>
```
Comment (mefarazath) [inline analytics-extensions/analytics.jsp:155]: new line

File: analytics-extensions/authenticationendpoint/localize.jsp [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+<%@ page import="java.nio.charset.StandardCharsets" %>
+<%@page contentType="text/html; charset=UTF-8"%>
+<%
+    String BUNDLE = "org.wso2.carbon.identity.application.authentication.endpoint.i18n.Resources";
+    ResourceBundle resourceBundle = ResourceBundle.getBundle(BUNDLE, request.getLocale(), new
+            EncodedControl(StandardCharsets.UTF_8.toString()));
+%>
+
+<%--Include analytics--%>
+<jsp:directive.include file="/includes/analytics.jsp"/>
```
Comment (mefarazath) [inline analytics-extensions/authenticationendpoint/localize.jsp:31]: new line

**Comment by ruwanta** [comment]
We need a README.md file explaining how to setup the sample.

**Pull request merged**